requirements/requirements-dev.txt

```diff
@@ -3,10 +3,13 @@ click==8.0.4
 docker==5.0.3
 dredd-hooks==0.2.0
 flake8==3.8.4
-mock==3.0.5
-model-bakery==1.5.*
+importlib-metadata<5  # need to pin <5 while flake8 is on 3.8.4
+mock==5.1.*
+model-bakery==1.13.*
 pre-commit==1.20.0
 pyspark==3.2.1
-pytest-cov==2.10.1
-pytest-django==4.2.0
-pytest==6.2.*
+pytest==7.4.*
```

Comment:
We have a requirements-test.txt now. Wouldn't it be better to have these test libraries added to that and that requirements-test.txt used?